recipes/forestci/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "forestci" %}
+{% set version = "0.1" %}
+{% set sha256 = "e706ec046c802c2e1036fd8ee306f1d9841b1a9239d0346f8c1849c5966578e0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - scikit-learn
+    - numpy
+    - nose
+
+test:
+  imports:
+    - forestci
+ 
+about:
+  home: https://github.com/scikit-learn-contrib/forest-confidence-interval
+  license: MIT
+  license_file: LICENSE.txt
```

Comment:
The `LICENSE.txt` file is not in the `MANIFEST.in`, so this will fail when trying to build from the pypi tarball since it won't be there. Comment out this line and add a commented note to add it once the`MANIFEST.in` has been patched upstream.